fmt/format.h

```diff
@@ -1331,11 +1331,11 @@ template <typename T, T> struct LConvCheck {
 // Returns the thousands separator for the current locale.
 // We check if ``lconv`` contains ``thousands_sep`` because on Android
 // ``lconv`` is stubbed as an empty struct.
-template <typename LConv>
+/*template <typename LConv>
 inline StringRef thousands_sep(
     LConv *lc, LConvCheck<char *LConv::*, &LConv::thousands_sep> = 0) {
   return lc->thousands_sep;
-}
+}*/
```

Comment:
This should be conditionally compiled instead of commented out.

Reply:
That was what commit eec3fcdeabc52f85fcb58a567fae0a6d5d700123 did :-)